xyz.eclipium.TetraSwap.json

```
@@ -0,0 +1,64 @@
+{
+    "app-id": "xyz.eclipium.TetraSwap",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "TetraSwap",
+	"rename-appdata-file": "TetraSwap.appdata.xml",
+	"rename-icon": "TetraSwap",
+	"rename-desktop-file": "TetraSwap.desktop",
+    "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--share=network"
+  	],
+    "modules": [
+        {
+            "name": "TetraSwap",
+            "buildsystem": "simple",
+            "build-commands": [
+            	"ls",
+            	"mkdir /app/bin",
+				"cp TetraSwap /app/bin/",
+				"cp -r TetraSwap_Data /app/bin/",
+				"cp UnityPlayer.so /app/bin/",
+				"mkdir /app/share",
+				"mkdir /app/share/applications",
+				"cp TetraSwap.desktop /app/share/applications/",
```

```suggestion
				"install -Dm644 TetraSwap.desktop /app/share/applications/${FLATPAK_ID}.desktop",
```

and remove the `rename-desktop-file` at the top